app/views/tasks/show.html.slim

```diff
@@ -478,13 +478,21 @@
             = button_tag class: 'btn btn-light dropdown-toggle', 'data-bs-toggle': 'dropdown' do
               = t('.button.add_to_collection')
             ul.scrollable.dropdown-menu[role="menu"]
-              - if !current_user.collections.empty?
-                - current_user.collections.reverse.each do |collection|
-                  li
-                    = link_to collection.title, add_to_collection_task_path(collection: collection), method: 'post', class: 'dropdown-item'
-              - else
+              - current_user.collections.reverse.each do |collection|
```

Comment:
I assume this is using `reverse` since collections have a default order by creation date, and we want to list the most recent first. Could be made explicit (but I saw that this code has been moved, and it was this way before).

Reply:
Good idea!